crates/dht/src/time.rs

```diff
@@ -200,6 +211,98 @@ impl PartitionedTime {
 
         Ok(())
     }
+
+    /// Inform the [PartitionedTime] that some ops have been stored.
+    ///
+    /// It is expected that the caller ensures that the ops belong to the hash range managed by this
+    /// [PartitionedTime]. This method will update the hashes of the full and partial slices that
+    /// the incoming ops belong to.
+    ///
+    /// If the op happens to be new enough to not belong in a slice, then it will be ignored. The
+    /// op will be discovered later when a partial slice update adds a slice that includes the op.
+    pub(crate) async fn inform_ops_stored(
```

Comment:
```suggestion
    /// op will be discovered later when a partial slice update adds a slice that includes the op.
```

Reply:
Not clear about this diff, possibly on the wrong line?

Reply:
No idea what happened here. I did this in the IDE probably and made a bollocks of it.